General/src/main/java/es/uclm/esi/isoft2/a04/Persistance/Broker.java

```diff
@@ -3,21 +3,41 @@
 import java.sql.*;
 import java.util.Vector;
 
+
 /**
- * @version 0.1.2
+ * @version 1.1.3
  *
```

Comment:
The version here should just increment the patch because it is a fix that does not affect the existing public side of the class